core/infra/docker-compose.yml

```diff
@@ -1,68 +1,30 @@
 services:
-  discovery-1:
-    container_name: core.discoveryprovider1.docker.co
+  sandbox-core:
+    container_name: sandbox-core
     build:
       context: ../
-      dockerfile: ./infra/Dockerfile
-      target: final
+      dockerfile: ./infra/Dockerfile.dev
     env_file:
-      - ./dev_config/discovery-one.docker.env
+      - ./dev_config/sandbox-one.env
     restart: unless-stopped
     ports:
       - "6610:26656" # CometBFT P2P Server
       - "6611:26657" # CometBFT RPC Server
       - "6612:50051" # Core GRPC Server
     networks:
-      - audius-protocol_default
+      - sandbox-core-net
 
-  content-1:
-    container_name: core.contentnode1.docker.co
-    build:
-      context: ../
-      dockerfile: ./infra/Dockerfile
-      target: final
-    env_file:
-      - ./dev_config/content-one.docker.env
-    restart: unless-stopped
-    ports:
-      - "6710:26656" # CometBFT P2P Server
-      - "6711:26657" # CometBFT RPC Server
-      - "6712:50051" # Core GRPC Server
-    networks:
-      - audius-protocol_default
-
-  content-2:
-    container_name: core.contentnode2.docker.co
-    build:
-      context: ../
-      dockerfile: ./infra/Dockerfile
-      target: final
-    env_file:
-      - ./dev_config/content-two.docker.env
-    restart: unless-stopped
-    ports:
-      - "6720:26656" # CometBFT P2P Server
-      - "6721:26657" # CometBFT RPC Server
-      - "6722:50051" # Core GRPC Server
-    networks:
-      - audius-protocol_default
-
-  content-3:
-    container_name: core.contentnode3.docker.co
-    build:
-      context: ../
-      dockerfile: ./infra/Dockerfile
-      target: final
-    env_file:
-      - ./dev_config/content-three.docker.env
-    restart: unless-stopped
+  postgres:
+    image: postgres:latest
+    container_name: sandbox-core-db
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: core_db
```

Comment:
gave sandbox it's own postgres and network so development doesn't interfere with audius-compose, if audius-compose is required you should write integration tests with the whole dev cluster